In [7]:
text = ['London Paris Paris London London London','Paris Paris London Paris']

In [8]:
from sklearn.feature_extraction.text import CountVectorizer
cv = CountVectorizer()
count_matrix = cv.fit_transform(text)

In [10]:
cv.get_feature_names()

['london', 'paris']

In [9]:
count_matrix.toarray()

array([[4, 2],
       [1, 3]], dtype=int64)

In [11]:
from sklearn.metrics.pairwise import cosine_similarity
similarity_scores = cosine_similarity(count_matrix)
print(similarity_scores)

[[1.         0.70710678]
 [0.70710678 1.        ]]


Text A and Text B are 70.71 % similar to each other